File: python/PostgreSQL.py
import psycopg2

hostname = "<$hostname>"
database = "<$database>"
port = "<$port>"
username = "<$user>"
password = "<$password>"

# Establishing the connection
conn = psycopg2.connect(
   database=database, user=username, password=password, host=hostname, port=port
)
# Creating a cursor object using the cursor() method
cursor = conn.cursor()

# Executing an MYSQL function using the execute() method
cursor.execute("select version()")

# Fetch a single row using fetchone() method.
data = cursor.fetchone()
print("Connection established to: ", data)

# Closing the connection
conn.close()

File: nodejs/PostgreSQL.js
const { Client } = require("pg");

const client = new Client({
  user: "<$user>",
  host: "<$hostname>",
  database: "<$database>",
  password: "<$password>",
  port: "<$port>",
});

async function main() {
  await client.connect();

  const res = await client.query("SELECT $1::text as message", [
    "Hello world!",
  ]);
  console.log(res.rows[0].message); // Hello world!
  await client.end();
}

main().catch(console.error);
